src/reservations/service/ReservationService.ts

```diff
@@ -1,14 +1,35 @@
-import { Reservation } from "../interface/ReservationInterface"
-import { ReservationRequestDto } from "../interface/ReservationInterfaceDto"
+import { Reservation } from "@api/reservations/interface/ReservationInterface";
+import { ReservationRequestDto } from "@api/reservations/interface/ReservationInterfaceDto";
+import { getDatabase } from "@api/storage/db";
 
 interface ReservationFilters {
   date?: string
 }
 
 export class ReservationService {
 
-  static create(reservation: ReservationRequestDto): Reservation {
-    throw new Error("Method not implemented.")
+  static async create(reservationDto: ReservationRequestDto): Promise<Reservation> {
+    try {
+      const bookingInfo = reservationDto.bookingInfo
+      const type = 'reservation'
+      const db = await getDatabase()
+      if (db) {
+        const reservations = db.collection('reservations')
```

Comment:
`const reservations = db.collection<Reservation>('reservations')`
Should be like that, but it'll complain a lot :) We can fix that another time

Reply:
well collection är ingen reservation utan en "tabell" av reservations. Plus att jag tror inte typescript fattar det då det är egentligen ett mongodb objekt.